moveit_plugins/moveit_ros_control_interface/moveit_core_plugins.xml

```diff
@@ -1,12 +1,12 @@
 <class_libraries>
   <library path="moveit_ros_control_interface_plugin">
-    <class name="moveit_ros_control_interface/MoveItControllerManager"
-           type="moveit_ros_control_interface::MoveItControllerManager"
+    <class name="moveit_ros_control_interface/Ros2ControlManager"
```

Comment:
I think you need to add the following lines; otherwise it will break current setups
 
```xml
<class name="moveit_ros_control_interface/MoveItControllerManager"
       type="moveit_ros_control_interface::Ros2ControlManager"
       base_class_type="moveit_controller_manager::MoveItControllerManager">
    <description>ros_control controller manager interface for MoveIt</description>
</class>
<class name="moveit_ros_control_interface/MoveItMultiControllerManager"
       type="moveit_ros_control_interface::Ros2ControlMultiManager"
       base_class_type="moveit_controller_manager::MoveItControllerManager">
    <description>multiple ros_control controller managers interface for MoveIt</description>
</class>
```

Reply:
We're thinking about this differently. My thought is, we change the plugin name and force users to adapt. There probably aren't many users out there using `MoveItControllerManager` on Rolling. Maybe just your client and PickNik.

The name `MoveItControllerManager` is misleading so I think we should stop using it.

Reply:
Please disregard my previous comment. I understand now.